basiclibrary/lib/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+import java.util.Random;
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    // Rolling Dice
+    public int[] roll(int n) {
+        Random rand = new Random();
+        int[] diceRoll = new int[n];
+        for (int i = 0; i < n; i++) {
+            int randNum = rand.nextInt(6);
```

Comment:
This code would produce a result from 0 to 6. Then, adding one would change the range to 1-7. The goal was to roll a 6 sided die so refactor to fix!